helm-icons.el

```diff
@@ -116,12 +125,23 @@ NAME, CLASS and ARGS are the original params."
        result)))
     result))
 
+(defun helm-icons--setup ()
+  "Setup icons based on which provider is set."
+  (cond ((eq helm-icons-provider 'all-the-icons)
+         (require 'all-the-icons)
+         (when (not (member "all-the-icons" (font-family-list)))
+         (all-the-icons-install-fonts)))
```

Comment:
Typically, the user is supposed to call that.

Reply:
It prompts the user when `(helm-icons-enable)` is run which saves another install step, and the majority of current users are likely to already have them installed.